build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -82,18 +85,25 @@ jobs:
   #       -BuildFlavor ${{ parameters.buildFlavor }}
 
   #   displayName: 'build-nupkg.ps1'
+  - task: PowerShell@2
+    displayName: UpdateFoundationNuSpecVersion
+    inputs:
+      targetType: 'inline'
+      script: |
+        $versionFinal = '${{ parameters.major }}.${{ parameters.minor }}.${{ parameters.patch }}-${{ parameters.versionTag }}.${{ parameters.versionDate }}.${{ parameters.versionCounter }}'
```

Comment:
do you enver need to worry abou thte "tag/date/counter" empty case (e.g. "release" versioning scheme?)

Reply:
I was just planning on setting it to “release” or “official” if that’s okay :)